src/main/java/com/cloudbees/jenkins/plugins/BitbucketJobProbe.java

```diff
@@ -81,14 +83,38 @@ private boolean match(SCM scm, URIish url) {
         if (scm instanceof GitSCM) {
             for (RemoteConfig remoteConfig : ((GitSCM) scm).getRepositories()) {
                 for (URIish urIish : remoteConfig.getURIs()) {
-                    if (GitStatus.looselyMatches(urIish, url))
+                    if (GitStatus.looselyMatches(urIish, url)) {
                         return true;
+                    }
+                }
+            }
+        } else if (scm instanceof MercurialSCM) {
+            try {
+                URI hgUri = new URI(((MercurialSCM) scm).getSource());
+                String remote = url.toString();
+                if (almostMatches(hgUri, remote)) {
```

Comment:
Yeah, I saw that when I was developing this, and I remember I couldn't use it.  I tried implementing it again and realized why I included the almostMatches method...  The 
[MercurialStatus.looselyMatches](https://github.com/jenkinsci/mercurial-plugin/blob/17a01f2097e3a90568b42705c9a0d223695bdef5/src/main/java/hudson/plugins/mercurial/MercurialStatus.java#L61) method is not public unfortunately.